packages/paste-website/src/components/shortcodes/PageAside.tsx

```diff
@@ -17,9 +18,11 @@ interface PageAsideProps {
     headings?: ToCHeading[];
   };
   hideFeedback?: boolean;
+  stickyTop?: BoxProps['top'];
+  topPadding?: BoxProps['padding'];
 }
 
-const PageAside: React.FC<PageAsideProps> = ({data, hideFeedback}) => {
+const PageAside: React.FC<PageAsideProps> = ({data, hideFeedback, stickyTop = 'space130', topPadding = 'space0'}) => {
```

Comment:
I'll blame the GitHub code reviews for obscuring this away. I'm setting overrides for these props on the implementation in the TokensList component - you can see the full file [here](https://github.com/twilio-labs/paste/blob/839c34380ca5571169fad9efaad810701c17e1ca/packages/paste-website/src/components/tokens-list/index.tsx#L144). Since GH only shows the code lines changed, and since the props are spread on multiple lines, the GH review UI isn't showing the full context of the change.